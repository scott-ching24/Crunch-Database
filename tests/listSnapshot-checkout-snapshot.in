set a 1 2
snapshot
set b 2 3
set c 3 4
snapshot
set a 3 4
checkout 1
list snapshots
list entries
bye
