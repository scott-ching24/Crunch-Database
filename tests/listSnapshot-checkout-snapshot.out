> ok

> saved as snapshot 1

> ok

> ok

> saved as snapshot 2

> ok

> ok

> 2
1

> a [1 2]

> bye
